gufe/tests/test_alchemicalnetwork.py

```diff
@@ -47,3 +47,22 @@ def test_connectivity(self, benzene_variants_star_map):
             else:
                 edges = alnet.graph.edges(node)
                 assert len(edges) == 0
+
+    def test_connected_subgraphs(self, benzene_variants_star_map):
```

Comment:
could you add a brief test that checks that this works for fully connected graph? i.e. it just returns itself

Reply:
yes good idea. I rearranged the fixtures a bit so we can access either one or both of the ligand and complex star networks for the benzene test case. I think this is the clearest way to do this without duplicating too much code.